nx/source/services/fs.c

```diff
@@ -633,6 +700,29 @@ Result fsIsExFatSupported(bool* out)
     return rc;
 }
 
+// Wrapper(s) for fsCreateSaveDataFileSystemBySystemSaveDataId.
+Result fsCreate_SystemSaveDataWithOwner(FsSaveDataSpaceId SaveDataSpaceId, u64 saveID, u128 userID, u64 ownerId, u64 size, u64 journalSize, u32 flags) {
+    FsSave save;
+    FsSaveCreate create;
+
+    memset(&save, 0, sizeof(save));
```

Comment:
This can be simplified by just doing:
```c
FsSave save = {
    .userID = userID,
    .saveID = saveID,
};
FsSaveCreate create = {
    .size = size,
    .journalSize = journalSize,
    .blockSize = 0x4000,
    .ownerId = ownerId,
    .flags = flags,
    .saveDataSpaceId = saveDataSpaceId,
};
```